identity-aware-proxy/IAPClient/IAPClient.cs

```diff
@@ -17,49 +17,45 @@
 // [START iap_make_request]
 
 using Google.Apis.Auth.OAuth2;
-using System.IO;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Threading;
 using System.Threading.Tasks;
 
-namespace GoogleCloudSamples
+public class IAPClient

```

Comment:
Are you not using namespaces at all for samples then? (I can see how that's appealing. Just thought I'd check.)

Reply:
(Noted later: IapClient may be better. That affects loads of code.)

Reply:
FWIW, I've been omitting the namespaces for new code samples I've been adding. Normally you'd want namespaces for every class, obviously, but I suppose the reasoning "it makes them easier to copy and paste" is sensible. Newer versions of C# seems to favor less and less syntax e.g. no need for brackets in using statements that are disposed at the end anyways, no need for a main method... seems reasonable?